app/views/users/show/pr/_pr.html.erb

```diff
@@ -1,25 +1,59 @@
-<tr>
```

Comment:
I'm really confused by the changes to this file -- shouldn't this just match master, with like a one-line change to pass the new `pr_type` through to the icon partial?